sites/page-details/index.json-ld

```diff
@@ -112,21 +112,30 @@
 				{
 					"en": { "href": "demoted/page-details-v3-en.html", "text": "Demoted - Page details version 3.0" },
 					"fr": { "href": "demoted/page-details-v3-fr.html", "text": "Rétrogradé - Détails de la page version 3.0" }
+				},
+				{
+					"en": { "href": "demoted/page-details-v4-en.html", "text": "Demoted - Page details version 3.1" },
+					"fr": { "href": "demoted/page-details-v4-fr.html", "text": "Rétrogradé - Détails de la page version 3.1" }
 				}
 			],
 			"implementation": [
 				"_:implement_pd",
 				"_:implement_pd_gcweb",
-				"_:implement_pd_3to4"
+				"_:implement_pd_3to4",
+				"_:implement_pd_4to5"
 			],
 			"history": [
+				{
+					"en": "July 2024 - Removed \"col-lg-9\" class.",
+					"fr": "Juillet 2024 - Retrait de la classe «&nbsp;col-lg-9&nbsp;»."
+				},
 				{
 					"en": "June 2024 - Removed \"Share this page\" option.",
 					"fr": "Juin 2024 - Retrait de l'option « Partagez cette page »."
 				},
 				{
-					"en": "March 2023 - Tag <footer> changed to <section>.",
-					"fr": "Mars 2023 - Balise <footer> changée pour <section>."
+					"en": "March 2023 - Tag <code>&lt;footer&gt;</code> changed to <code>&lt;section&gt;</code>.",
+					"fr": "Mars 2023 - Balise <code>&lt;footer&gt;</code> changée pour <code>&lt;section&gt;</code>."
```

Comment:
Revised this part to prevent ``"isHTML": true`` from causing ``<footer>`` and ``<section>`` to disappear.

Btw it doesn't look like any of GCWeb's other history notes wrote element tags "as-is" like that. So it shouldn't be a problem anywhere else.

Reply:
Thank you!